lib/Agrammon/UI/CommandLine.pm6

```diff
@@ -118,6 +123,26 @@ sub USAGE() is export {
     USAGE
 }
 
+sub create-user($cfg-filename, $username, $firstname, $lastname, $password, $role?) {
+    get-cfg-and-db-handle($cfg-filename);
+    my $user = Agrammon::DB::User.new(
+        :$username, :$firstname, :$lastname, :$password
+    );
+    $user.create-account($role);
+    CATCH {
+        when X::Agrammon::DB::User::Exists  {
+            note "User $username already exists";
+            return;
```

Comment:
I do wonder if we should exit with a non-zero error code here (in case somebody decides to use this in a script)...

Reply:
Ok, will add one.